ui/src/components/config_section/index.scss

```diff
@@ -11,7 +11,7 @@
 .euiPanel--configSection {
   .euiDescriptionList.euiDescriptionList--responsiveColumn {
     > * {
-      @include euiTextTruncate;
+      @include euiTextBreakWord;
```

Comment:
@krithika369, I can't seem to override this mixin that truncates quite a number of text descriptions in other UI components - I tried creating custom style rules for the `Container` table but they don't seem to be able override this mixin either :/

Reply:
Oh we shouldn't change it globally for all rows.

> I tried creating custom style rules for the Container table

Could you share the example here? What was tried and where it was placed?

Reply:
So I tried creating this custom style class and using it for the `EuiDecriptionList` of a `ContainerConfigTable`, but it doesn't overwrite any of the truncation styles:

```scss
.euiDescriptionList--responsiveColumn.containerConfig {
  .euiDescriptionList__description {
    @include euiTextBreakWord;
  }
}
```

Explicitly specifying the style of the `euiTextBreakWord` mixin  as found [here](https://github.com/elastic/eui/blob/491840ed37e6b0bba0f731ef848fe45326e41e6b/src/global_styling/mixins/_typography.scss#L127) doesn't seem to work either:

```scss
.euiDescriptionList--responsiveColumn.containerConfig {
  .euiDescriptionList__description {
    overflow-wrap: break-word !important;
    word-wrap: break-word !important;
    word-break: break-word;
  }
}
```

Reply:
Another thing I've tried to try was to add `<div className="eui-textBreakWord">` tags to the `description` of the item corresponding to the image name within the `items` list of the `ContainerConfigTable` as:

```js
{
  title: "Image",
  description: (
      <div className="eui-textBreakWord">
        thisisaverylargexcepetionnamewithalotofwordsthatatsomepointaregoingtobetruncated
      </div>
    )
}
```
Although this doesn't 'truncate' the line and replace the missing characters with an ellipses, it doesn't break the line either... generating something like this instead: 
![Screenshot 2022-03-30 at 10 59 51 AM](https://user-images.githubusercontent.com/36802364/160742406-b718679e-c7c0-482f-9a9b-d1efa5ea2249.png)

Reply:
I see.. I was going to suggest something along the lines of your second example to override the properties for the single row's description. But adding the `div` is probably messing with some other flex properties. I will try to have a stab at it to understand better.

Reply:
Thanks a lot! I'll see what I can try out from my end!

Reply:
Using `span` in combination with inline style overrides seems to do it:
```javascript
const items = [
    {
      title: "Image",
      description: (<span style={{
        wordBreak: "break-word",
        textOverflow: "unset",
        whiteSpace: "break-spaces"
      }} >
        {image}
      </span >),
    },
```

We're having to override some existing `!important` properties which doesn't seem easy with classes.

Reply:
Wow thanks for discovering that! I'll include your code snippet in!